src/App.js

```diff
@@ -1,19 +1,44 @@
 import React from 'react';
+import { useState } from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
+
+const chatDataList = chatMessages
```

Comment:
Since you have `import chatMessages from './data/messages.json';` on line 5, you're getting the data from a file and referring to that data as `chatMessages`. You now can directly use chatMessages without needing to create a new variable chatDataList and using the second variable to reference it. 

This works just like on line2 where you import `useState` then use the function in this file `useState('some initial value')`